### DISCUSSION TITLE
XCH farmer reward not received

### DISCUSSION THREAD

**Discussion by xbrsx** [Chia-Network/chia-blockchain#5859] [Support]
Hello,

I can see through Chia Explorer that I have won farmer reward yesterday and, accordingly, received 2 XCH. However, till now I have not received anything on my wallet and the gui shows all of my XCH balance, XCH Total Chia Farmed and XCH Block Rewards as Zero. Knowing that the full node and my wallet are completely "Synced". I have even tried to restart the gui, reinstall, and still now seeing anything.

Is there any issue? or Am I missing something?

![image](https://user-images.githubusercontent.com/84662622/119256065-71f99480-bbc7-11eb-84e9-009bf4e1d5ae.png)

![image](https://user-images.githubusercontent.com/84662622/119256076-82aa0a80-bbc7-11eb-852c-1b0ab39451be.png)

![image](https://user-images.githubusercontent.com/84662622/119256046-5e4e2e00-bbc7-11eb-8bc5-f2fae10909a3.png)

![image](https://user-images.githubusercontent.com/84662622/119255976-0c0d0d00-bbc7-11eb-993b-881558435f5c.png)

Edit: 

I noticed that my wallet address is different from the Farmer/Pool reward address. I read it's fine, but to double check, is this an issue?

![image](https://user-images.githubusercontent.com/84662622/119259464-1e8f4280-bbd7-11eb-9525-54c03218cb84.png)

![image](https://user-images.githubusercontent.com/84662622/119256000-2e9f2600-bbc7-11eb-9f4f-dd583ba11d6c.png)

**Comment by BrandtH22** [comment]
@xbrsx , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

Note - if you are still experiencing the issue feel free to reopen this ticket or join us in discord. The rewards addresses should be the first wallet addresses while the other one listed in the gui could be a latter address that connects to the same wallet

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
